src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CollectionExpression.cs

```diff
@@ -56,20 +62,34 @@ private BoundExpression RewriteCollectionExpressionConversion(Conversion convers
             }
         }
 
+        private BoundExpression VisitImmutableArrayCollectionExpression(BoundCollectionExpression node, TypeSymbol elementType)
+        {
```

Comment:
Please add pseudo code for the generated IL

Reply:
Gentle ping

Reply:
I believe I've addressed this now, thanks!